src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
+spring.datasource.url= jdbc:mysql://db.live-poll.de:3306/livepoll
```

Comment:
We should extract connection urls, usernames, and first of all passwords to environment variables to ensure security. See [here](https://github.com/ChilliBits/particulate-matter-api/blob/develop/src/main/resources/application.yml#L15) for an example.